packages/calcite-components/src/components/navigation-user/navigation-user.scss

```diff
@@ -43,23 +43,40 @@
 }
 
 calcite-avatar {
-  @apply px-4;
+  --calcite-avatar-corner-radius: var(--calcite-navigation-user-avatar-corner-radius);
+  --calcite-avatar-color: var(--calcite-navigation-user-avatar-color);
+  padding-inline: 1rem;
 }
 
 calcite-avatar ~ .text-container {
   @apply ps-0;
 }
 
 .full-name {
-  @apply text-0
-  ms-0
-  text-color-1
-  font-medium;
+  @apply ms-0 text-0 font-medium;
+  color: var(--calcite-navigation-user-full-name-text-color, var(--calcite-color-text-1));
 }
 
 .username {
-  @apply text-color-2;
   font-size: var(--calcite-font-size--1);
+  color: var(--calcite-navigation-user-name-text-color, var(--calcite-color-text-2));
+}
+
+:host(:hover) .button,
+:host(:focus) .button {
+  --calcite-internal-navigation-user-background-color: var(--calcite-color-foreground-2);
+}
+
+:host(:focus) .button {
+  @apply focus-inset;
+}
+
+:host(:active) .button {
+  --calcite-internal-navigation-user-background-color: var(--calcite-color-foreground-3);
```

Comment:
Not entirely sure here but I think this can be simplified.

Currently the pattern looks like
navigation.background.color = color.foreground.1
:hover,:focus = color.foreground.2
:active = color.foreground.3

but the background color was already set at the parent component "navigation". The hover press states here aren't actually trying to change the color but imply a shadow like the button is indenting. Therefor we can update this to use the color.transparent pattern for hover,active state.

remove line 28
remove lines 65 - 80

replace with

```css
:host .button {
  background-color: var(--calcite-color-transparent);

  &:hover,
  &:focus {
    background-color: var(--calcite-color-transparent-hover);
  }

  &:focus {
    @apply focus-inset;
  }

  &:active {
    background-color: var(--calcite-color-transparent-press);
  }
}

:host([active]) .button {
  --calcite-internal-navigation-selection-color: var(--calcite-color-brand);
}
```

The background is always whatever the navigation.background.color is.